rnd/docker-compose.yml

```diff
@@ -24,11 +49,9 @@ services:
     networks:
       - app-network
 
-  rest_server:
-    build:
-      context: ../
-      dockerfile: rnd/autogpt_server/Dockerfile
-      target: server
+  api_srv:
```

Comment:
please change this in the AGENTSERVER_HOST as well if you change the name here. However, I think server is more clear than srv